morse_decode.rb

```diff
@@ -0,0 +1,48 @@
+@morse = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode(message)
+  # split the message into an array or words
+  words = message.split('   ')
+  decoded_message = ''
+
+  words.each do |word|
+    # split each word into an array or letters
+    letters = word.split
+    letters.each do |letter|
+      # Take each letter and check it in the morse dictionary
+      # and add it to the decoded message
+      decoded_message += @morse[letter]
+    end
+    decoded_message += ' '
+  end
+  print decoded_message
+end
```

Comment:
- [ ] You have only one method that does all deciphering, but you should have three (3).

  - The first method should decode ONLY ONE morse character.
  - The second method should decode a morse word using the first method.
  - The last method should decode a sentence using the second method.

  Kindly implement that. Ensure that the methods **`return`**, the decoded characters/words not **`print`** them so they can used in subsequent methods